iam/api/AccessTest/AccessTest.cs

```diff
@@ -36,19 +36,21 @@ public AccessTest()
             _member3 = "daeneryssnow.827847@gmail.com";
         }
 
-        [Fact]
-        public void TestAccess()
+        public IamService initializeService()
         {
-            // Initializing service for role creation
             var credential = GoogleCredential.GetApplicationDefault()
                 .CreateScoped(IamService.Scope.CloudPlatform);
             var service = new IamService(new IamService.Initializer
             {
                 HttpClientInitializer = credential
             });
 
-            // Create custom roles for testing
-            var role1 = new Role
+            return service;
+        }
+
+        public Role createCustomRole(IamService service)

```

Comment:
public methods must start with capital letters in C#

Reply:
Done

Reply:
Done